packages/java-packages/snippet-replacer-maven-plugin/src/main/java/azuresdk/plugin/SnippetDictionary.java

```diff
@@ -0,0 +1,38 @@
+package azuresdk.plugin;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SnippetDictionary {
+    private HashMap<String, List<String>> _snippetDictionary = new HashMap<String, List<String>>();
+
+    public SnippetDictionary(){
+
+    }
```

Comment:
empty ctor can be removed

Reply:
I use the empty constructor in a test though, when I remove it that crashes. Is there something specific I need to do to enable that?

Reply:
In the absence of any constructor, a default public constructor is provided. I'm not sure why you would see a crash without this, but having said that, it is unlikely to be worth your time investigating so you should probably just leave as-is.

Reply:
(this is after a swap to Java 8).

I removed default ctor, now see this:

![image](https://user-images.githubusercontent.com/45376673/80432011-de607200-88a7-11ea-8b64-533733ab2d7e.png)

![image](https://user-images.githubusercontent.com/45376673/80432020-e3252600-88a7-11ea-8e0f-85638caa97e0.png)

I think it's because I provide a non-default one WITH arguments as well.